### PR TITLE
Birmingham Class 2 - Nirmeet - Week 2 - Corrections made in homework.md

### PR DESCRIPTION
**Pull request by nirmeet-baweja**
The errors in the file homework.md have been corrected to the best of my knowledge.

# Your Details

- Your Name: Nirmeet Kaur Baweja
- Your City: Wolverhampton
- Your Slack Name: Nirmeet Kaur Baweja

# Homework Details

- Module: Git & Github
- Week: 2

### CODE REVIEW

**Review by jwillson2020 — COMMENTED** [review]
Great answers Nirmeet,  they all look right and well explained,  ;-)

**Comment by hiqbal0121** [comment]
Looks good Nirmeet! Some really good explanations also.

File: homework.md [hunk 1]
```diff
@@ -92,19 +95,19 @@ A place for us to store and share our code.
...
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+Commit messages are important to give a brief description about the changes made and the features added. So, that whenever we come back to see the history, it helps to understand why a certain commit was made.
 
 ## Question 18
```
Comment (ATedy) [inline homework.md:113]: Nice work Nirmeet, brief and detailed.

Comment (remidoukmak) [reply homework.md:113]: This looks good and well explained.

**Pull request closed without merging**